Optimal_bounding_box/include/CGAL/Optimal_bounding_box/internal/nelder_mead_functions.h

```diff
@@ -50,7 +50,7 @@ Matrix mean(const Matrix& m1,
 {
   typedef typename Traits::FT                                 FT;
 
-  constexpr FT half = FT(1) / FT(2);
```

Comment:
That is because we use `-frounding-math` with gcc: that tells the compiler that floating-point arithmetic may be evaluated, at runtime, with a modified rounding-mode. And thus gcc thinks it cannot evaluate the expression at compile-time.

Note also that, if `FT` is a class, `FT(1)/FT(2)` can be only be a `constexpr` if:
  - the constructor of `FT` from `int` is `constexpr`,
  - and the `operator/` is also `constexpr`.

For an `FT` with dynamic allocation (like `CGAL::Gmpq`, or `CGAL::Lazy`), there is no chance that it can be `constexpr`.